src/proxy/router.js

```diff
@@ -93,6 +95,10 @@ export default class Router {
         return false;
     }
 
+    _processStaticContent () {
+        // Do nothing =)
+    }
```

Comment:
In the dom processor we are using 'throw new Error('Not implemented');' for the abstract methods.